tools/generate_csv/generate_csv.py

```diff
@@ -7,14 +7,48 @@
 import urllib.request, json
 from urllib.request import urlopen, Request
 from SPARQLWrapper import SPARQLWrapper, JSON
+from concurrent.futures import ThreadPoolExecutor
+from os import makedirs
+from typing import List
+from pathlib import Path
+<<<<<<< HEAD
```

Comment:
I guess these merge artifacts are not intended for commit? :-)

Reply:
indeed my bad